molgenis/client.py

```diff
@@ -1,6 +1,5 @@
 import json
```

Comment:
Please fix all the warnings in this file (including the Sonar warning)

Reply:
Done, except from the one in the "get" method. That would require me to remove options to pass in your API, which is undesirable.

Reply:
Alright, and I see we can't suppress Sonar warnings in PyCharm. That's too bad.

There's two warnings left still: the cognitive complexity one hasn't been resolved yet and there's an unknown import warning at the top (I've added a GitHub suggestion to fix that one).

Also there's three small warnings left in `client_test` (some semicolons and an unused variable). I know these ones probably aren't new but they're easy to fix.

Reply:
> > > The tutorial.py file can't be run because of JSON decoding errors
> 
> > Tutorial.py will not work anymore with the v2 version of the API because it links to a server which doesn't have the api/v2 yet. Should we remove it?
> 
> Yes let's do that. Are the other two examples sufficient as a tutorial?

I think they are a bit minimal. We may need a new one, but let's make another story for that.